docs/faq.rst

```diff
@@ -200,18 +200,25 @@ and call :meth:`aiohttp.web.WebSocketResponse.close` on all of them in
         web.run_app(app, host='localhost', port=8080)
 
 
-How do I make a request from a specific IP address?
----------------------------------------------------
+How do I make a request from a specific IP address or network interface?
+------------------------------------------------------------------------
 
 If your system has several IP interfaces, you may choose one which will
-be used used to bind a socket locally::
+be used used to bind a socket locally, either by specifying the source IP, or
+source network interface. It is also possible to have a
+fallback by catching the :exc:`RuntimeError` on creation::
+
+    try:
+        conn = aiohttp.TCPConnector(network_interface="eth0")
```

Comment:
Should we use the `local_addr` argument here too so it's closer to the fallback?